jpo-ode-core/src/main/resources/schemas/schema-bsm.json

```diff
@@ -499,17 +487,11 @@
                                                         "required": [
                                                             "eventHazardLights",
                                                             "eventStopLineViolation",
-                                                            "eventABSactivated",
-                                                            "eventTractionControlLoss",
-                                                            "eventStabilityControlactivated",
                                                             "eventHazardousMaterials",
                                                             "eventReserved1",
                                                             "eventHardBraking",
                                                             "eventLightsChanged",
-                                                            "eventWipersChanged",
-                                                            "eventFlatTire",
-                                                            "eventDisabledVehicle",
-                                                            "eventAirBagDeployment"
+                                                            "eventWipersChanged"
                                                         ],
```

Comment:
We ought to modify the redaction functionality to set bitstring-associated boolean values to false instead of deleting them. I'll open a work item in the backlog.